policy/modules/contrib/ldap.te

```diff
@@ -77,9 +74,9 @@ manage_dirs_pattern(slapd_t, slapd_log_t, slapd_log_t)
 manage_files_pattern(slapd_t, slapd_log_t, slapd_log_t)
 logging_log_filetrans(slapd_t, slapd_log_t, { file dir })
 
-manage_dirs_pattern(slapd_t, slapd_replog_t, slapd_replog_t)
-manage_files_pattern(slapd_t, slapd_replog_t, slapd_replog_t)
-manage_lnk_files_pattern(slapd_t, slapd_replog_t, slapd_replog_t)
+manage_dirs_pattern(slapd_t)
+manage_files_pattern(slapd_t)
+manage_lnk_files_pattern(slapd_t)
 
```

Comment:
line 77, 78,and 79 can be removed as those patterns expect 3 parameters and it wont build with 1 parameter.

Reply:
Thanks, SElinux policies are a bit new to me.